browser/metrics/variations_browsertest.cc

```diff
@@ -38,13 +38,12 @@ const SignedSeedData& GetBraveSignedSeedData() {
       "xmQaBThr1i8ARQ1rKEinHluXeR7ve5fqy7L4AIgNym2PbtlL+9142+"
       "T8gUjjEsoT28J3HqE4IEa1eFvKLw=";
 
-  static const SignedSeedData kBraveTestSeedData{study_names,
-                                                 kBase64UncompressedData,
-                                                 kBase64CompressedData,
-                                                 kBase64Signature,
-                                                 /*in_compressed_data=*/{},
-                                                 /*in_compressed_data_size=*/0};
-  return kBraveTestSeedData;
+  return {study_names,
+          kBase64UncompressedData,
+          kBase64CompressedData,
+          kBase64Signature,
```

Comment:
for this to work all `constexpr char` vars in this function should be marked `static`, otherwise it'll trip sanitizer.